lib/cangallo/qcow2.rb

```diff
@@ -45,6 +45,12 @@ def self.qemu_img_version
       end
     end
 
+    def self.qemu_img_backing_file_parameter(path)
+      q = Qcow2.new(path)
+      format = q.info['format']
```

Comment:
What if it's an unsupported format?

Reply:
I've added a new test that creates backing files with different supported formats. If I am not mistaken if the format is not supported it returns raw format.